elixir/lib/contracts.ex

```diff
@@ -15,13 +15,20 @@ defmodule Trento.Contracts do
     id = Keyword.get(opts, :id, UUID.uuid4())
     source = Keyword.get(opts, :source, "trento")
     data = Protobuf.Encoder.encode(struct)
+    seconds = Keyword.get(opts, :seconds, :os.system_time(:second))
 
     cloud_event =
       CloudEvents.CloudEvent.new(
         data: {:proto_data, Google.Protobuf.Any.new!(value: data, type_url: get_type(mod))},
         spec_version: "1.0",
         type: get_type(mod),
         id: id,
+        attributes: %{
+          "ce_timestamp" =>
```

Comment:
Maybe just `timestamp` here?

Reply:
this should be called `time` actually, since `ce_timestamp` is just the name of the `oneof` field

Reply:
fair enough